runtime/common/ServerHelper.h

```diff
@@ -59,7 +59,7 @@ class ServerHelper : public registry::RegisteredType<ServerHelper> {
   virtual const std::string name() const = 0;
 
   /// @brief Set the number of shots to execute
-  void setShots(std::size_t s) { shots = s; }
+  void set_shots(std::size_t s) { shots = s; }
```

Comment:
Hmm. All the other member functions in this class use camelCase.

Reply:
True - hopefully it's pybind but ill improve this just in case